### PR TITLE
Fix NPE in JakartaEE9Action#transformApp

### PR DESCRIPTION
**Pull request by volosied**
Defect 290531 (iSeries):

Should I catch the NPE or just log the information for now to debug it further? 

Should something also be ported over to JakartaEE10Action?

```
java.lang.NullPointerException: Null output stream
at java.io.PrintStream.requireNonNull(PrintStream.java:103)
at java.io.PrintStream.<init>(PrintStream.java:175)
at java.io.PrintStream.<init>(PrintStream.java:159)
at componenttest.rules.repeater.JakartaEE9Action.transformApp(JakartaEE9Action.java:375)
at componenttest.rules.repeater.JakartaEE9Action.transformApp(JakartaEE9Action.java:358)
at componenttest.rules.repeater.JakartaEE9LooseAppsRule.lambda$featureWabTransform$5(JakartaEE9LooseAppsRule.java:258)
at componenttest.rules.repeater.JakartaEE9LooseAppsRule$$Lambda$115/0x3383c128.accept(Unknown Source)
at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:195)
at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:186)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:204)
at java.util.Iterator.forEachRemaining(Iterator.java:127)
at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1812)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:524)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:514)
at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:162)
at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:185)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:245)
at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:429)
at componenttest.rules.repeater.JakartaEE9LooseAppsRule.featureWabTransform(JakartaEE9LooseAppsRule.java:251)
at componenttest.rules.repeater.JakartaEE9LooseAppsRule.before(JakartaEE9LooseAppsRule.java:66)
at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:115)
```

### CODE REVIEW

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
ISERIES Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_pxPFMJbTEe2YL83b0ePHDw

File: dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java [hunk 1]
```diff
@@ -365,13 +365,18 @@ protected static void transformApp(Path appPath, Path newAppPath, Map<String, St
...
+        } catch (Exception e) {
+            Log.info(c, m, "Transformer Log Failure: " + e.getCause() + " " + e.getMessage());
+        }
+
+        if(fos != null){
+            PrintStream ps = new PrintStream(fos);
+            System.setOut(ps);
+            System.setErr(ps);
+        } else {
+            Log.info(c, m, "fos is null. Log not created for " +  appPath.getFileName() );
```
Comment (jhanders34) [inline dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java]:
```suggestion
            PrintStream ps = new PrintStream(fos);
            System.setOut(ps);
            System.setErr(ps);
        } catch (Exception e) {
            Log.info(c, m, "Transformer Log Failure for " +  outputLog, e);
```

**Review by jhanders34 — CHANGES_REQUESTED** [review]
fos.close() later in this method will also need a null check if fos can possibly be null.

**Comment by volosied** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#build

**Pull request merged**